From e520142aaa123822e728d8d65a194ec95569a4be Mon Sep 17 00:00:00 2001
From: Alexey Minnekhanov <alexeymin@postmarketos.org>
Date: Tue, 11 Feb 2025 12:23:55 +0300
Subject: [PATCH] dt-bindings: arm: qcom: Add Xiaomi Mi Pad 4 / 4 Plus

Xiaomi Mi Pad 4 (xiaomi,clover) is a 8-inch tablet based on SDA660
SoC. Xiaomi Mi Pad 4 Plus (xiaomi,clover-plus) is a 10-inch tablet
based on SDM660 SoC.

Signed-off-by: Alexey Minnekhanov <alexeymin@postmarketos.org>

Source-Commit: https://github.com/sdm660-mainline/linux/commit/e520142aaa123822e728d8d65a194ec95569a4be
Scope: Clover-specific
Porting notes: Only the LTE compatible used by the packaged DTB is retained; add it to the reorganized Linux 7.3 binding.
---
diff --git a/Documentation/devicetree/bindings/arm/qcom.yaml b/Documentation/devicetree/bindings/arm/qcom.yaml
--- a/Documentation/devicetree/bindings/arm/qcom.yaml
+++ b/Documentation/devicetree/bindings/arm/qcom.yaml
@@ -936,6 +936,7 @@
 
       - items:
           - enum:
+              - xiaomi,clover-lte
               - xiaomi,lavender
           - const: qcom,sdm660
 
